lib/s3_storage.rb

```diff
@@ -31,6 +31,10 @@ def public_url_for(asset)
     object_for(asset).public_url(virtual_host: AssetManager.aws_s3_use_virtual_host)
   end
 
+  def presigned_url_for(asset)
+    object_for(asset).presigned_url(:get, expires_in: 1.minute, virtual_host: AssetManager.aws_s3_use_virtual_host)
```

Comment:
Minor: It would be nice to pull the `expires_in` option value into the application configuration c.f. `AssetManager.aws_s3_use_virtual_host`.